src/modules/mount/main.py

```diff
@@ -149,6 +149,21 @@ def get_btrfs_subvolumes(partitions):
 
     return btrfs_subvolumes
 
+def enable_swap_partition():
+    swap_is_exist=subprocess.check_output(["swapon","-s"], universal_newlines=True)
```

Comment:
Don't use `subprocess`, instead use the built-in Calamares functions as described in the modules readme.